src/EnergyPlus/DemandManager.cc

```diff
@@ -666,7 +667,7 @@ namespace DemandManager {
         using DataHeatBalance::ZoneElectricObjects;
         using DataZoneControls::TempControlledZone;
         using DataZoneControls::TStatObjects;
-        using ExteriorEnergyUse::ExteriorLights;
+        //!$using ExteriorEnergyUse::ExteriorLights;
```

Comment:
remove since we are passing in the state struct instead of extern global variables,